package-lock.json

```diff
@@ -29,6 +29,28 @@
         "js-tokens": "^4.0.0"
       }
     },
+    "@babel/parser": {
```

Comment:
You didn't add any packages to the package.json. Did you mean to update and add dependencies?